resources/lib/sosac.py

```diff
@@ -196,8 +231,10 @@ def list_episodes(self, url):
             for series_key, episode in series.iteritems():
                 for episode_key, video in episode.iteritems():
                     item = self.video_item()
-                    item['title'] = series_key + "x" + episode_key + " - " + video['n']
-                    item['img'] = IMAGE_EPISODE + video['i']
+                    item['title'] = series_key + "x" + \
+                        episode_key + " - " + video['n']
```

Comment:
Nie je toto zbytocne zalomene ? Nezda sa mi, ze bez toho by bol ten riadok pridlhy. Taktiez, ak chces vyskladavat zlozitejsie stringy, tak radsej pouzivaj `.format()`. Mozes aj `%`, ale to je deprecated a ja by som to nepouzival.